viaq-jenkins/fields.yml

```diff
@@ -0,0 +1,493 @@
+          description: >
+            The name of the normalizer
+
+      - name: "@version"
+        index: not_analyzed
+        type: string
+        example: TODO
+        description: >
+          Version of “com.redhat.viaq” mapping the document is intended to adhere by the normalizer. It must be set by the normalizer. 
+          The value must correspond to the [_meta][version].
+
+      - name: original_raw_message
+        type: string
+        description: >
+          The original non-parsed log message, collected by collector or as close to the source as possible.
+          The field is analyzed for now for troubleshooting purposes. In future we may change it to not_analyzed
+
+      - name: trace
+        type: string
+        analyzer: whitespace
+        example: "shipshift,<version>,2016.02.01 logstash,1.5,2016.03.03"
+        description: >
+          The field records the trace of the message.
+          Each collector/normalizer appends information about itself and the date/time when the message was processed.
+
+    - shipshift_metadata:
+      type: group
+      name: shipshift_metadata
+      description: >
+        Placeholder for shipshift metadata. In control by shipshift team.
+      fields:
+
+    - CI_master:
+      type: group
+      description: >
+        Metadata related to the master server of CI (f.e. Jenkins master).
+      name: ci_master
+      fields:
+      - name: hostname
+        type: string
+        index: not_analyzed
+        description: >
+          FQDN of the CI master.
+
+    - CI_agent:
+      type: group
+      description: >
+        Metadata related to the CI agent/slave that executed the job.
+      name: ci_agent
+      fields:
+      - name: label
+        type: string
+        index: not_analyzed
+        description: >
+          TODO.
+
+      - name: name
+        type: string
+        index: not_analyzed
+        description: >
+          TODO.
+
+    - CI_job:
+      type: group
+      description: >
+        Metadata related to the CI job itself.
+      name: ci_job
+      fields:
+      - name: name
+        type: string
+        index: not_analyzed
+        description: >
+          Name of the CI job.
+
+      - name: build_id
+        type: integer
+        description: >
+          Build ID of the CI job.
+
+      - name: phase
+        type: string
+        index: not_analyzed
+        description: >
+          Phase of the CI job.
+
+      - name: status
+        type: string
+        index: not_analyzed
+        description: >
+          CI job status.
+
+      - name: log_url
+        type: string
+        index: not_analyzed
+        description: >
+          URL to the html log of the CI job.
+
+      - name: full_url
+        type: string
+        index: not_analyzed
+        description: >
+          Full URL to the CI job on the CI master.
+
+    - name: file
+      type: string
+      index: not_analyzed
+      description: >
+        name of the log file. 
```

Comment:
Optional field present when the data is collected from a file from a known file location.
The field may contain the full local path to the file.